### ISSUE TITLE
Unexpected char

### ISSUE THREAD

**Issue by pouljohn1** [tapglue/android_sdk#18]
```
java.lang.IllegalArgumentException: Unexpected char 0x15b at 5 in header value: Czas środkowoeuropejski standardowy
                                                                                       at com.squareup.okhttp.Headers$Builder.checkNameAndValue(Headers.java:295)
                                                                                       at com.squareup.okhttp.Headers$Builder.add(Headers.java:245)
                                                                                       at com.squareup.okhttp.Request$Builder.addHeader(Request.java:197)
                                                                                       at com.tapglue.networking.TGNetworkManager$1.intercept(TGNetworkManager.java:181)
                                                                                       at com.squareup.okhttp.Call$ApplicationInterceptorChain.proceed(Call.java:232)
                                                                                       at com.squareup.okhttp.logging.HttpLoggingInterceptor.intercept(HttpLoggingInterceptor.java:204)
                                                                                       at com.squareup.okhttp.Call$ApplicationInterceptorChain.proceed(Call.java:232)
                                                                                       at com.squareup.okhttp.Call.getResponseWithInterceptorChain(Call.java:205)
                                                                                       at com.squareup.okhttp.Call.access$100(Call.java:35)
                                                                                       at com.squareup.okhttp.Call$AsyncCall.execute(Call.java:171)
                                                                                       at com.squareup.okhttp.internal.NamedRunnable.run(NamedRunnable.java:33)
                                                                                       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
                                                                                       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
                                                                                       at java.lang.Thread.run(Thread.java:818)
```

**Comment by panzerdev** [comment]
What value is that and why does it even go into the header?

**Comment by dlsniper** [comment]
That error is triggered by the timezone identifier. This will be fixed in 1.1.
It goes in the header of the first request only as it's used to provide various analytics functionality.
